modules/custom-post-types/portfolios.php

```diff
@@ -713,6 +723,19 @@ static function get_project_tags( $post_id ) {
 	}
 
 	/**
+	 * Displays the author of the current portfolio project.
+	 *
+	 * @return html
+	 */
+	static function get_project_author() {
+		$html = '<div class="project-author"><span>' . esc_html__( 'Author:', 'jetpack' ) . '</span> ';
```

Comment:
@jeherve indeed, I had initially thought about writing this block as:
```php
$html = '<div class="project-author">';
$html .= sprintf( __( '<span>Author:</span> <a href="%s">%s</a>', 'jetpack' ),
	esc_url( get_author_posts_url( get_the_author_meta( 'ID' ) ) ),
	esc_html( get_the_author() )
);
$html .= '</div>';
```
but I went with the current format to write a markup similar to the one used to output project types and project tags.
What do you think? Personally, I think this one works better.